Cargo.toml

```diff
@@ -27,8 +27,10 @@ tonic = { version = "0.8.0", optional = true }
 prost = { version = "0.11.0", optional = true }
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread"], optional = true }
 uuid = { version = "1.1.2", features = ["serde", "v4"], optional = true }
-bincode = { version = "1.3.3", optional = true }
+bincode = { version = "1.3.3" }
 futures-util = "0.3.21"
+sev = { version = "1.0.0", optional = true, git = "https://github.com/larrydewey/virtee-sev.git", branch = "43-Clean-Extended-Report" }
+raw-cpuid = "10.6.0"
```

Comment:
should `raw-cpuid` be `optional = true` as well, and listed below as a dependency for specific feature `cc_kbc` ?